src/replication.c

```diff
@@ -2494,6 +2509,7 @@ char *sendCommand(connection *conn, ...) {
     while (1) {
         arg = va_arg(ap, char *);
         if (arg == NULL) break;
+        if (strcmp(arg, "") == 0) continue;
```

Comment:
A few things things:
1. Empty strings can be valid arguments, this might break some obscure functionality like if someone set a password like an empty string. (I don't think that is allowed today, but would like to be more resistant to weird edge cases. You can introduce an arbitrary static string that points to nothing to have the same behavior.
2. If you continue with this path, you should use strncmp() where possible. You can actually just do `if (arg[0] == '\0')` as well.

Reply:
IMO using some const indicator is not the correct way. Can we just use SendCommandArgv?